releases/release-1.35/release-notes/maps/pr-134298-map.yaml

```
@@ -0,0 +1,6 @@
+pr: 134298
+releasenote:
+  text: "Set `failCgroupV1` will be set to true from 1.35. \nThis means that nodes
+    will not start on a cgroup v1 by default. \nThis is putting cgroup v1 into a deprecated
+    state."
```

Prior reviews have included the `ACTION REQUIRED` part - https://github.com/kubernetes/sig-release/pull/2845/commits/de504120aaa1f6b189fb95722afcc77e6d62bfbd

```suggestion
  text: "ACTION REQUIRED: `failCgroupV1` will be set to true from 1.35. \nThis means that nodes
    will not start on a cgroup v1 by default. This puts cgroup v1 into a deprecated
    state."
```

---

Although I'm not sure if the sentence itself should begin with ACTION REQUIRED...

---

I do see one instance of ACTION REQUIRED in the 1.28 release notes.